tests/core/test_addasset.py

```diff
@@ -0,0 +1,74 @@
+import os
+from tempfile import TemporaryDirectory
+from unittest import TestCase
+
+import pystac
+
+from stactools.core import add_asset_to_item
+from tests import test_data
+from tests.utils import create_temp_copy
+
+
+class AddAssetTest(TestCase):
+    def test_add_asset_to_item(self):
+        with TemporaryDirectory() as tmp_dir:
+            item_path = create_temp_copy(
+                test_data.get_path("data-files/core/simple-item.json"),
+                tmp_dir,
+                "item.json",
+            )
```

Comment:
Since we're not moving the asset, do we need to create a copy of the item in this unit test?